packages/notification-service/src/firebase.ts

```diff
@@ -239,10 +259,10 @@ export async function sendPaymentNotification(
   console.info(NOTIFICATIONS_TAG, 'Block delay: ', lastBlockNotified - blockNumber)
   const t = getTranslatorForAddress(recipientAddress)
   data.type = NotificationTypes.PAYMENT_RECEIVED
-  const isCeloReward = celoRewardsSenders.indexOf(senderAddress) >= 0
+  const { title, body } = notificationTitleAndBody(senderAddress, currency)
   return sendNotification(
-    t(isCeloReward ? 'rewardReceivedTitle' : 'paymentReceivedTitle'),
-    t('paymentReceivedBody', {
+    t(title),
+    t(body, {
```

Comment:
Nothing to do now but I would love for us to move to extracting locale strings automatically from the code base. Declaring it this way makes it impossible for such extraction to happen.

Reply:
This sounds interesting but I don't fully understand. What would happen after we detect a new locale string on the code base? Is this related to Crowdin?

Reply:
Not directly related to Crowdin, but instead of manually maintaining the strings like we do today, there are tools that can extract them and build the final file containing all strings in use. This way unused strings get remove and new strings added and developers just need to run the extractions (or part of CI).
Usually, the condition for these tools to work is to have all strings be declared statically.